doc/plugins/error-log-logger.md

```diff
@@ -0,0 +1,101 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+- [中文](../zh-cn/plugins/error-log-logger.md)
+
+# Summary
+
+- [**Name**](#name)
+- [**Attributes**](#attributes)
+- [**How To Enable And Disable**](#how-to-enable-and-disable)
+- [**How To Update**](#how-to-update)
+
+## Name
+
+`error-log-logger` is a plugin which push the log data of APISIX's error.log to TCP servers.
+
+This will provide the ability to send the log data which selected by the level to Monitoring tools and other TCP servers.
+
+This plugin provides the ability to push the log data as a batch to you're external TCP servers. In case if you did not recieve the log data don't worry give it some time it will automatically send the logs after the timer function expires in our Batch Processor.
```

Comment:
`you're` => your?
`recieve` => `receive`

Reply:
:), fix it